src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSpawnCommand.java

```diff
@@ -43,45 +50,22 @@ public MythicSpawnCommand() {
     // -->
 
     @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("location")
-                    && arg.matchesArgumentType(LocationTag.class)) {
-                scriptEntry.addObject("location", arg.asType(LocationTag.class));
-            }
-            else if (!scriptEntry.hasObject("level")
-                    && arg.matchesPrefix("level", "l")
-                    && arg.matchesInteger()) {
-                scriptEntry.addObject("level", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("name")) {
-                scriptEntry.addObject("name", arg.asElement());
-            }
-            else {
-                arg.reportUnhandled();
-            }
-        }
-        if (!scriptEntry.hasObject("location") || !scriptEntry.hasObject("name")) {
-            throw new InvalidArgumentsException("Must specify a name and location.");
-        }
-        scriptEntry.defaultObject("level", new ElementTag(1));
+    public void addCustomTabCompletions(TabCompletionsBuilder tab) {
+        tab.add((Set<String>) MythicMobsBridge.getMobManager().getMobNames());
```

Comment:
Should remove the `Set<String>` cast here, as these methods take an `Iterable` now